base/statistics.jl

```diff
@@ -335,12 +335,18 @@ unscaled_covzm(x::AbstractMatrix, y::AbstractMatrix, vardim::Int) =
 # covzm (with centered data)
 
 covzm(x::AbstractVector; corrected::Bool=true) = unscaled_covzm(x) / (_length(x) - Int(corrected))
-covzm(x::AbstractMatrix, vardim::Int=1; corrected::Bool=true) =
-    scale!(unscaled_covzm(x, vardim), inv(size(x,vardim) - Int(corrected)))
+function covzm(x::AbstractMatrix, vardim::Int=1; corrected::Bool=true)
+    C = unscaled_covzm(x, vardim)
+    T = typeof(first(C) / 1)
```

Comment:
It can. Are you worried about `first` not being representative of the elements in `C`?